kubernetes/kfserving/examples/mnist_readme.md

```diff
@@ -17,11 +17,16 @@ To serve an Inference Request for Torchserve using the KFServing Spec, follow th
 inference_address=http://127.0.0.0:8085
 management_address=http://127.0.0.0:8081
 number_of_netty_threads=4
-service_envelope=kfserving
+enable_envvars_config=true
```

Comment:
Please verify this does not have impact on GCAIP integration which relies on V1 spec at present